package.json

```diff
@@ -12,7 +12,7 @@
     "globby": "^8.0.0",
     "google-auto-auth": "^0.9.0",
     "google-proto-files": "^0.15.0",
-    "grpc": "~1.9.1",
+    "grpc": "^1.10.0",
```

Comment:
Still uncertain about ^ vs ~: https://github.com/googleapis/nodejs-common-grpc/pull/23#discussion_r175755409

Reply:
OK to use `^` for me.